compiler/ballerina-lang/src/main/java/io/ballerina/projects/Package.java

```diff
@@ -490,15 +490,19 @@ private Package createNewPackage() {
                 CompilerContext compilerContext = project.projectEnvironmentContext()
                         .getService(CompilerContext.class);
                 PackageCache packageCache = PackageCache.getInstance(compilerContext);
+
+                io.ballerina.projects.environment.PackageCache service =
```

Comment:
Here `service` is not a descriptive variable name

Reply:
Changed from `service` to `environmentPackageCache`